Impossible-Finance-Agents/Swap-Fees/src/agent.ts

```diff
@@ -12,18 +12,18 @@ import PairFetcher from "./pairs.fetcher";
 const FACTORY: string = "0x86f9944711526af414683033E5846E92b721191A"; // PoC Factory in Goerli
```

Comment:
Since other bots were updated to monitor V3 factory, this one needs to be updated too. 

```suggestion
const FACTORY: string = "0x4233ad9b8b7c1ccf0818907908a7f0796a3df85f"; // V3 Swap Factory
```